apiserver/pkg/chat/chat.go

```diff
@@ -102,8 +103,8 @@ func AppRun(ctx context.Context, req ChatReqBody, respStream chan string) (*Chat
 	if err != nil {
 		return nil, err
 	}
-	klog.Infoln("begin to run application", obj.GetName())
-	out, err := appRun.Run(ctx, c, respStream, application.Input{Question: req.Query, NeedStream: req.ResponseMode == Streaming, History: conversation.History})
+	klog.FromContext(ctx).Info("begin to run application", "appName", req.APPName, "appNamespace", req.AppNamespace)
```

Comment:
klog.V(5) ?

Reply:
Info or V(5).Info are both ok for me.
The update inherited from the original of @nkwangleiGIT .
IMO, this log conveniently allows us to calculate how much time was actually spent running the app, leaving aside the chat api initialization time.

Reply:
That's good. 

Seems to me 

```shell
V(5)  ->  `DEBUG` 
V(3) ->  Info
V(1) ->  ERROR
```

Since we are going to release v0.1.0 , it's the time that we should make a agreement on logging levels.

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use

* `klog.InfoS` -  Structured logs to the INFO log. `InfoS` should be used for routine logging. It can also be used to log warnings for expected errors (errors that can happen during routine operations).
  Depending on log severity it's important to pick a proper verbosity level to ensure that consumer is neither under nor overwhelmed by log volume:
  * `klog.V(0).InfoS` = `klog.InfoS` - Generally useful for this to **always** be visible to a cluster operator
    * Programmer errors
    * Logging extra info about a panic
    * CLI argument handling
  * `klog.V(1).InfoS` - A reasonable default log level if you don't want verbosity.
    * Information about config (listening on X, watching Y)
    * Errors that repeat frequently that relate to conditions that can be corrected (pod detected as unhealthy)
  * `klog.V(2).InfoS` - Useful steady state information about the service and important log messages that may correlate to significant changes in the system.  This is the recommended default log level for most systems.
    * Logging HTTP requests and their exit code
    * System state changing (killing pod)
    * Controller state change events (starting pods)
    * Scheduler log messages
  * `klog.V(3).InfoS` - Extended information about changes
    * More info about system state changes
  * `klog.V(4).InfoS` - Debug level verbosity
    * Logging in particularly thorny parts of code where you may want to come back later and check it
  * `klog.V(5).InfoS` - Trace level verbosity
    * Context to understand the steps leading up to errors and warnings
    * More information for troubleshooting reported issues